plugin/adsk/scripts/mayaUSDRegisterStrings.mel

```diff
@@ -120,7 +120,7 @@ global proc mayaUSDRegisterStrings()
     register("kTimeAnn", "Edits the current time value of a stage, which corresponds to the animation frame drawn in the viewport. By default, this value connects to Maya's global time node.");
     register("kTipYouCanChooseMultipleFiles", "<b>Tip:</b>  You can choose multiple files.");
     register("kUniversalSceneDescription", "Universal Scene Description");
-    register("kUsdFileOptions", "USD File Options");
+    register("kUsdFileOptions", "Stage");
```

Comment:
In these cases I also like to rename the kUsdFileOptions variable (make it more descriptive of the string) 
 since it forces us to look at every place this string is used and ensure we aren't changing a place we don't intend to.